app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const { getTopics } = require('./controllers/topicsController')
+console.log(getTopics)
+const app = express();
+
+app.get('/api/topics', getTopics)
+
+module.exports = app;
+
+app.use((err, req, res, next) => {
+    console.log(err);
+    res.status(500).send('Server Error!');
+  });
+
+app.all('/*', (req, res) => {
```

Comment:
I would maybe move this app.all to be above the app.use on line 10, just to keep the endpoints together and the error handling at the bottom of the file for readability :)